near-contract-standards/src/non_fungible_token/mod.rs

```diff
@@ -12,6 +12,9 @@ mod macros;
 /// Metadata traits and implementation according to the [NFT enumeration standard](https://nomicon.io/Standards/NonFungibleToken/Metadata.html).
 /// This covers both the contract metadata and the individual token metadata.
 pub mod metadata;
+
+pub mod payout;
```

Comment:
Module isn't documented here or in `.../payout/mod.rs`,

can you describe this and link to the spec there using `//!` rustdoc comments at the top of the file, please?